core/test/oracle/Governor.js

```diff
@@ -799,15 +804,23 @@ contract("Governor", function(accounts) {
 
   it("startingId size", async function() {
     // Starting id of 10^18 is the upper limit -- that should be the largest that will work.
-    await Governor.new(Finder.address, toWei("1"), Timer.address, { from: proposer });
+    await Governor.new((await Finder.deployed()).address, toWei("1"), (await Timer.deployed()).address, {
+      from: proposer
+    });
 
     // Anything above 10^18 is rejected.
-    assert(await didContractThrow(Governor.new(Finder.address, toWei("1.1"), Timer.address, { from: proposer })));
+    assert(
+      await didContractThrow(
+        Governor.new((await Finder.deployed()).address, toWei("1.1"), (await Timer.deployed()).address, {
```

Comment:
The nesting is kinda hard to read here, what about instantiating the contract instances first and then passing in `finder.address` and `timer.address` instead?